### PR TITLE
Player Movement

### PR DESCRIPTION
**Pull request by ChaseBENNC9**
Basic Player Movement has been added
Controls are W (Into screen)
A right
S towards screen
D left

The Cube is a just a visual placeholder. 
Speed can be adjusted in the inspector to find a value that works the best.

### CODE REVIEW

File: AI Game Jam/Assets/Scripts/PlayerMovement.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerMovement : MonoBehaviour
+{
+    private CharacterController controller;
+    private Vector3 playerVelocity;
+    private bool groundedPlayer;
+    private float playerSpeed = 2.0f;
```
Comment (erokasyuart) [inline AI Game Jam/Assets/Scripts/PlayerMovement.cs]: 4 feels a lot more controllable.

**Review by erokasyuart — CHANGES_REQUESTED** [review]
Looks good, just feels a bit slow so I played around with a speed I liked. Great job on the invisible wall.

File: AI Game Jam/Assets/Scripts/PlayerMovement.cs (hunk 1 above)
Comment (ChaseBENNC9) [reply AI Game Jam/Assets/Scripts/PlayerMovement.cs]: Changed default speed to 4

**Review by erokasyuart — APPROVED** [review]
Much better! I am ready for this to be merged into staging!

**Pull request merged**